app/src/main/java/com/mealsmadeeasy/data/FakeMealStore.kt

```diff
@@ -2,12 +2,13 @@ package com.mealsmadeeasy.data
 
 import com.mealsmadeeasy.model.*
 import io.reactivex.Observable
+import io.reactivex.Single
 import io.reactivex.subjects.BehaviorSubject
 import org.joda.time.DateTime
 
 class FakeMealStore : MealStore {
 
-    private val mealPlan: BehaviorSubject<MealPlan>
+    private var mealPlan: BehaviorSubject<MealPlan>
```

Comment:
This should still be a val. The idea of observables is that anyone who is subscribed can see changes, but if you change the observable then you can't retain the original subscribers. So if you put values into this subject with `Subject.onNext(T)`, then anything that's still subscribed will see the changes. (TL;DR If you do it this way then the UI won't automatically update)